### PR TITLE
Add OWNERS file to define initial set of approvers

### PR DESCRIPTION
**Pull request by k4leung4**
and reviewers

/cc bbrowning, greghaynes

### CODE REVIEW

**Comment by houshengbo** [comment]
@k4leung4 
I am not quite sure of what reviewers mean.
I mean I can be a reviewer. Anyone can be reviewer.

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+approvers:
+- bbrowning
+- greghaynes
+- jcrossley3
+- kleung
+- evankanderson  # TOC member as backup
+- mattmoor       # TOC member as backup
+
+reviewers:
```
Comment (houshengbo) [inline OWNERS]: What do reviewers mean?

Comment (k4leung4) [reply OWNERS]: You are correct that anyone can be a reviewer.
The reviewer section is used by the bots to auto-assign/suggest reviewers. 

This is to ensure that the TOC members that are backup approvers are not spammed by review requests.

If you like, I can add you you to the reviewer section.

Comment (houshengbo) [reply OWNERS]: I definitely want to be one.

Comment (k4leung4) [reply OWNERS]: added

Comment (yu2003w) [reply OWNERS]: @k4leung I also would like to be reviewer.  Could you please add me to the list?

Comment (k4leung4) [reply OWNERS]: @yu2003w added.

**Pull request merged**